modules/aws-emr-ports/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "master_ports" {
```

Comment:
Prior versions of the module took into account which applications you were running, with support for hbase, spark, and ganglia, to decide which ports to create network rules for.

We'll need to change strategies here to keep parity with how the module used to behave. Otherwise, we're potentially creating rules for ports we don't need.

I like that we're outputting just two lists of ports, one for master and one for core, because that seems easy to use.

So maybe in this nested ports module if we took boolean variables like spark, hbase, ganglia, we could create those output dynamically.

That might involve breaking up master_ports into `master_ports_emr` `master_ports_spark` `master_ports_hbase` `master_ports_ganglia` so that we can use the booleans to add those into the return 1:1.

Reply:
done